index.js

```diff
@@ -79,7 +79,7 @@ var Sqlite3Driver = Base.extend({
       constraint.push('UNIQUE');
     }
 
-    if (spec.defaultValue) {
+    if (spec.defaultValue !== undefined) {
```

Comment:
Using `undefined` ensures that if you set a `defaultValue`, it will get set. Currently, it's not predictable. The user _could_ set the defaultValue to `null`, but so what? 

Maybe we filter out that case specifically instead of naming out all the falsey values? Thoughts?

```js
if (spec.defaultValue !== undefined && spec.defaultValue !== null)
```